ml-conversational-analytic-tool/tests/test_commentAnalysis.py

```diff
@@ -0,0 +1,64 @@
+import unittest
+
+from ..commentAnalysis import CommentAnalyzer
```

Comment:
```suggestion
import commentAnalysis
```

Reply:
Not sure why those ".." are included.  But it is much preferred to import modules, not classes.